pkgs/tools/filesystems/fuse-7z-ng/default.nix

```diff
@@ -14,6 +14,9 @@ stdenv.mkDerivation rec {
     # Drop unused pthread library. pthread_yield()
     # fails the configure.
     ./no-pthread.patch
+    # Zero-initialize unset fields of `struct fuse_operations` so that
+    # garbage values don't cause segfault.
```

Comment:
I suggest linking https://github.com/kedazo/fuse-7z-ng/pull/8 right into this comment. That way it will be easier to track the fate of the patch by others.